src/components/replay.js

```diff
@@ -0,0 +1,24 @@
+AFRAME.registerComponent("replay", {
+  init: function() {
+    this.playhead = 0;
+    this.poseIndex = 0;
+  },
+
+  tick: function(t, dt) {
+    let overflow = false;
+    while (!overflow && this.playhead >= this.poses[this.poseIndex].timestamp) {
+      this.el.setAttribute("position", this.poses[this.poseIndex].position);
+      this.el.setAttribute("rotation", this.poses[this.poseIndex].rotation);
```

Comment:
We can avoid calling `setAttribute` here but I didn't know if the `rotation` euler needed to be a component so that its `order` was changed. I decided to just copy what the original code did. We can address it later if we need to optimize the bot code (which I doubt will be especially important anytime soon).

Reply:
Yeah don't think bot perf matters to much